{"uuid": "6ef0d925-3836-43c6-8259-126690790783", "children": ["d4f20c44-766f-4628-8120-8dbcd03fce53", "58793d72-b719-4490-a6a1-b70ecc0cd5e6", "07f39a88-ec1b-400f-9abc-2bf0067b70c3"], "befores": [{"name": "init_driver", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'web_driver' referenced before assignment\n", "trace": "  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\SauravSharma\\Pytest\\Pom-Pytest\\Tests\\conftest.py\", line 12, in init_driver\n    request.cls.driver = web_driver\n"}, "start": 1648475887352, "stop": 1648475887353}], "start": 1648475887352, "stop": 1648475887379}